daemon/cmld.c

```diff
@@ -1607,18 +1618,12 @@ cmld_container_stop(container_t *container)
 
 	DEBUG("Trying to stop container %s", container_get_description(container));
 
-	if (container_get_state(container) == COMPARTMENT_STATE_STOPPED) {
-		ERROR("Container %s not running, unable to stop",
-		      container_get_description(container));
-
-		audit_log_event(container_get_uuid(container), FSA, CMLD, CONTAINER_MGMT,
-				"container-stop-failed", uuid_string(container_get_uuid(container)),
-				0);
-		return -1;
-	}
-
 	int ret = container_stop(container);
```

Comment:
use container_is_stoppable here instead of changing return value API of compartment_stop(), see comment in compartment.c